telegram/ext/_application.py

```diff
@@ -1672,3 +1688,64 @@ async def process_error(
 
         _logger.exception("No error handlers are registered, logging exception.", exc_info=error)
         return False
+
+
+class BaseProcessor:
+    def __init__(self, max_concurrent_updates: int):
+        self.max_concurrent_updates = max_concurrent_updates
```

Comment:
I suggest to make this a property. That way it's clear that this value can not be changed at runtime (which it can't be anyways since `self.semaphore` wouldn't reflect those changes)